cmake/projects/odb-sqlite/hunter.cmake

```diff
@@ -21,6 +22,7 @@ hunter_add_version(
     3be07e7702abf8adcbe7736f372ef9980cec1003
 )
 
+hunter_configuration_types(odb-sqlite CONFIGURATION_TYPES Release)
 hunter_pick_scheme(DEFAULT url_sha1_odb-sqlite_autotools)
 hunter_download(
```

Comment:
Also I guess I missed that before: probably there is no need for `PACKAGE_DEPENDS_ON`